pocs-capstone/frontend/src/components/PetDisplay/PetDisplay.js

```diff
@@ -36,6 +36,36 @@ const HAPPY = 'H'
 const TASK_URL = "/tasks/"
 const USER_URL = "/user-data/"
 const TODAY = new Date()
+
+const WEIGHTS = [
+    '15 paperclips',
+    '3 nickels',
+    '5 pieces of bubblegum',
+    '4 mullberries',
+    '2/3 a medium radish',
```

Comment:
``` 2'3 of a medium radish ```

Reply:
this does sound better ty

Reply:
thanks to this I also noticed "23/4 a slice of multigrain bread" should have "of"